### PR TITLE
L-Space Stuff; this time with fewer conflicts.

### PR DESCRIPTION
**Pull request by NigglyWiggly**
Just the same shit as the last one

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 1]
```diff
@@ -44,7 +44,7 @@
 /mob/living/simple_animal/crab/attackby(var/obj/item/O as obj, var/mob/user as mob)
 	if(istype(O, /obj/item/weapon/wirecutters))
 		if(prob(50))
-			user << "<span class='danger'>This kills the crab.</span>"
+			user << "<span class='warning'><B>This kills the crab.</span></B>"
```
Comment (ComicIronic) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: This is not how to resolve conflicts correctly.

**Comment by clusterfack** [comment]
Triggered

File: code/modules/mob/living/simple_animal/friendly/crab.dm (hunk 1 above)
Comment (clusterfack) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: hahaha

Comment (NigglyWiggly) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: I didn't actually intend to change anything there, fug.

File: code/modules/mob/living/simple_animal/friendly/crab.dm [hunk 2]
```diff
@@ -44,7 +44,6 @@
 /mob/living/simple_animal/crab/attackby(var/obj/item/O as obj, var/mob/user as mob)
 	if(istype(O, /obj/item/weapon/wirecutters))
 		if(prob(50))
-			user << "<span class='danger'>This kills the crab.</span>"
```
Comment (ComicIronic) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: AUGH

Comment (NigglyWiggly) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: Goddammit What did I fuck up this time

Comment (NigglyWiggly) [inline code/modules/mob/living/simple_animal/friendly/crab.dm]: Oh god how did I even do that

**Pull request merged**